troupon/static/scss/partials/_typography.scss

```diff
@@ -43,3 +43,30 @@ h1 {
     margin-right: 5px;
   }
 }
+
+.fa-edit {
+  padding-left: 8px;
+}
+
+.item-info {
+  .title, .advertiser, .price, .address {
+    padding-bottom: 20px;
+  }
+
+  address {
+    margin-bottom: 0px
+  }
+
+  .quorum {
+    margin-top: 30px;
+  }
+
+  a:hover {
+    cursor: pointer
+  }
+
+  .label-text {
+    color: #5E5E77;

```

Comment:
Color literals like `#5E5E77` should only be used in variable declarations; they should be referred to via variable everywhere else.<br>Color `#5E5E77` should be written as `#5e5e77`